data/settings.js

```diff
@@ -10453,4 +10453,451 @@ If enabled, protocol-level SSL errors are logged. Same as
+Set to \`yes\` to use TLS to connect to the LDAP server.`
+	},
+
+	ssl_ca_cert_dir: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+Specify a value for TLS \`ssl_ca_cert_dir\` option.
+
+Currently supported only with OpenLDAP.`
+	},
+
+	ssl_ca_cert_file: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+Specify a value for TLS \`ssl_ca_cert_file\` option.
+
+Currently supported only with OpenLDAP.`
+	},
+
+	ssl_cert_file: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+Specify a value for TLS \`ssl_cert_file\` option.
+
+Currently supported only with OpenLDAP.`
+	},
+
+	ssl_cipher_suite: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+Specify a value for TLS \`ssl_cipher_suite\` option.
+
+Currently supported only with OpenLDAP.`
+	},
+
+	ssl_key_file: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+Specify a value for TLS \`ssl_key_file\` option.
+
+Currently supported only with OpenLDAP.`
+	},
+
+	ssl_require_cert: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.ENUM,
+		values_enum: [ 'never', 'hard', 'demand', 'allow', 'try' ],
+		text: `
+Specify a value for TLS \`ssl_require_cert\` option.
+
+Currently supported only with OpenLDAP.`
+	},
+
+	ldap_uris: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+LDAP URIs to use.
+
+Configure this setting to specify what LDAP
+server(s) to connect to.
+
+Note that this setting isn't supported by all LDAP libraries.
+
+The URIs are in syntax \`protocol://host:port\`.
+
+Example: \`ldap_uris = ldaps://secure.domain.org\``
+},
+
+fields: {
+	tags: [ 'ldap' ],
+	default: '<empty>',
+	values: setting_types.STRING,
+	text: `
+	FIXME: SEE ABOVE
+::: info
+	LDAP Authentication Only
+:::
+
+Specify user attributes to be retrieved from LDAP (in userdb look up).
+
+User attributes are given in \`dovecot-internal-name=%{ldap:LDAP-name}\` lines.
+
+The internal names are:
+
+| Name | Description |
+| ---- | ----------- |
+| \`uid\` | System UID |
+| \`gid\` | System GID |
+| \`home\` | Home directory |
+| \`mail\` | [[link,mail_location]] |
+
+There are also other special fields which can be returned. See
+[[link,userdb_extra_fields]].
+
+Example:
+
+\`\`\`
+fields {
+	home = %{ldap:homeDirectory}
+	uid = %{ldap:uidNumber}
+	gid = %{ldap:gidNumber}
+}
+\`\`\``
+	},
+
+	ldap_filter: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+		#FIXME: duplicate
+::: info
+	LDAP Authentication Only
+:::
+
+Filter for user lookup (userdb lookup).
+
+Variables that can be used (see [[variable]] for full list):
+
+| Variable | Long Name | Description |
+| -------- | --------- | ----------- |
+| \`%u\` | \`%{user}\` | Username |
+| \`%n\` | \`%{username}\` | User part in user@domain, same as %u if there's no domain |
```

Comment:
I don't think we should duplicate this list, just extra work to update it when Aki soon removes the short variables anyway. Link to variables is enough.